yurt/orchestration/roles/db/tasks/main.yml

```diff
@@ -3,9 +3,9 @@
 - name: Install PostgreSQL
   apt: name={{ item }} update_cache={{ update_apt_cache }} state=installed
   with_items:
-    - postgresql=9.4*
-    - postgresql-contrib=9.4*
-    - python-psycopg2=2.5.*
+    - postgresql
```

Comment:
I think the issue with having Yurt do dependency pinning in this case is that if there's already a version of Postgresql on the server that doesn't match up with the one Yurt is installing, this step will be skipped (with `ok` status since Postgresql has already been installed) and then on a step after - the one that checks the config file at the specific version's directory - it would error out looking for the version that Yurt thought it installed, when it should just go for one that already exists.

The current approach ensures that Yurt is fine with currently existing versions of PostgreSQL. Alternatively, I can see if there's an option that will force `apt` to install the version we want if that version doesn't exist, but that could affect other projects on the same server.